style.css

```diff
@@ -0,0 +1,67 @@
+body {
+    
+    background-size: cover;
+
+    font-size: 20px;
+    margin: 0;
```

Comment:
In relation to linters, the 4-space indetion and empyty spaces can be eliminated, or caught by css(stylintrc) linters, and your code would look professional and nicer😃